examples/connectivity/plot_atlas_comparison.py

```diff
@@ -0,0 +1,57 @@
+"""
+Extracting signals from different atlases and compare correlation matrices
+==========================================================================
+
+In this example :class:`nilearn.input_data.NiftiLabelsMasker` is used to
+extract time series from nifti objects using different parcellation atlases.
+
+The time series of all subjects of the ADHD Dataset are concatenated to create
+parcel-wise correlation matrices for each atlas.
+
+"""
+
+from nilearn import datasets
+from nilearn.input_data import NiftiLabelsMasker
+import numpy as np
+from matplotlib import pyplot as plt
+
+# load atlases
+destrieux = datasets.fetch_atlas_destrieux_2009()
+destrieux_atlas = destrieux['maps']
+
+yeo = datasets.fetch_atlas_yeo_2011()
+yeo_atlas = yeo['thick_17']
+
+harvard_oxford = datasets.fetch_atlas_harvard_oxford('cort-maxprob-thr25-2mm')
+harvard_oxford_atlas = harvard_oxford['maps']
+
+atlases = {'Destrieux Atlas': destrieux_atlas,
+           'Yeo Atlas 17 thick': yeo_atlas,
+           'Harvard Oxford > 25%': harvard_oxford_atlas}
```

Comment:
To make the example shorter, I think that you can even avoid the variables 'destrieux_atlas', 'yeo_atlas', 'harvard_oxford_atlas' by directly putting 'destrieux['maps']', etc in the dictionary.